data.json

```diff
@@ -1,4 +1,4 @@
-[
+[A
```

Comment:
I guess this is a typo here :)

Reply:
Yes, that was a mistake - sorry! Should be fixed now!